scripts/extensions/markForUser/src/get-mark-for-user-modal.tsx

```diff
@@ -65,6 +65,7 @@ export function getMarkForUserModal(
                             disabled={locked}
                             onSelect={(selectedUser) => this.setState({selectedUserId: selectedUser._id})}
                             selectedUserId={this.state.selectedUserId}
+                            onFocus={{hideOptions: true}}
```

Comment:
Can't we make it so it's open by default and closed when the item is already marked for a user?

Reply:
Nope. Users want to be able to quickly select another user to mark for when they click the avatar or "mark for another user". I've added an option to `SelectUser` so you can choose whether it auto-expands or not depending on where you are rendering it.